.circleci/config.in.yml

```diff
@@ -251,5 +286,16 @@ workflows:
               only:
                 - master
     jobs:
+      - hold:
```

Comment:
It'd probably be good to make this into it's own separate workflow, so that it can also be triggered on PRs

Reply:
Do I need to add something else here?